### PR TITLE
relocate run methods from container to image

### PR DESCRIPTION
**Pull request by dhodovsk**
fixes: https://github.com/fedora-modularity/conu/issues/11

### CODE REVIEW

File: conu/apidefs/image.py [hunk 1]
```diff
@@ -118,6 +118,24 @@ def mount(self, mount_point=None):
         """
         raise NotImplementedError("mount is not implemented")
 
+    def run_via_binary(self, *args, **kwargs):
+        """
+        create container using image and run it in the background; this method is useful
```
Comment (TomasTomecek) [inline conu/apidefs/image.py]: "using (this|the) image"

File: conu/apidefs/image.py [hunk 2]
```diff
@@ -118,6 +118,24 @@ def mount(self, mount_point=None):
...
+        to test real user scenarios when users invoke containers using binary and not an API
+
+        :param image: instance of Image
+        :return: instance of Container
+        """
+        raise NotImplementedError("run_via_binary method is not implemented")
+
+    def run_via_api(self, container_params):
+        """
+        create container using image and run it in the background
```
Comment (TomasTomecek) [inline conu/apidefs/image.py]: the same

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/6u0Z](https://da.gd/6u0Z)__
* Comment: __See details for logs__

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/oRQB](https://da.gd/oRQB)__
* Comment: __See details for logs__

**Pull request merged**